tests/unit-tests/tests-templates.php

```diff
@@ -43,6 +43,12 @@ public function test_get_donation_form() {
 		$this->assertContains( '<form id="give-form-', $form );
 		$this->assertContains( 'class="give-form', $form );
 		$this->assertContains( 'method="post">', $form );
+		$this->assertContains( 'data-currency_position="', $form );
```

Comment:
@DevinWalker Add tests for data attributes

Reply:
I like it

Reply:
Can you also assert it matches the currency of the specific form?

Reply:
@DevinWalker Pr updated